server/dao/games.js

```diff
@@ -26,20 +42,61 @@ function getGame(id) {
  * @returns {string} - game id
  */
 function createGame(data) {
-  // data.fen (string) (game state) (eg: "k7/7R/8/8/8/8/8/5KR1 w - - 0 1")
-  // TODO:
+  // TODO: how to determine if userId is black or white?
+  const url = `http://localhost:8080/game`;
+
+  let config = {
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization' : data.token
+    }
+  }
+
+  let body = {
+    "whiteId": data.userIdOne,
+    "blackId": data.userIdTwo,
+    "state": data.fen
+  }
+
+	return axios.post(url, body, config)
+		.then(res => res.data)
+		.catch(e => {
+      console.error(`ERROR: Authorization token not valid!`)
+			throw new Error(`Authorization token not valid!`);
+    });
 }
 
 /**
  * update game
- * @param {string} id - game id
+ * @param {string} gameId - game id
  * @param {object} data - object containing game state as string (fen)
  * @returns {string} - game id
  */
-function updateGame(id, data) {
-  // TODO:
+function updateGame(gameId, data) {
+  // TODO: playerId in body not needed
+  const url = `http://localhost:8080/state/update`;
+
+  let config = {
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization' : data.token
```

Comment:
i suggest keeping a map with token per username and if it responds with 403 to refresh the map

Reply:
also we need the bearer

Reply:
Ako ima vreme sled nakraq 🙂